release_notes/ocp_3_6_release_notes.adoc

```diff
@@ -0,0 +1,2470 @@
+* Support for xref:../dev_guide/builds/build_strategies.adoc#docker-strategy-build-args[arguments in Docker files].
+* xref:../dev_guide/builds/build_strategies.adoc#jenkins-pipeline-strategy-environment[Environment variables in pipeline builds].
+* Credential support for Jenkins Sync plug-in for ease of working external Jenkins instance.
+* xref:../dev_guide/builds/build_environment.adoc#overview[`ValueFrom` Support] in build environment variables.
+* Deprecated Jenkins v1 image.
+* `oc cluster up`: support launching service catalog
+* Switch to nip.io from xip.io, with improved stability
+
+[[ocp-36-web-console]]
+=== Web Console
+
+[[ocp-36-service-catalog]]
+==== Service Catalog (Technology Preview)
+
+You can now opt into the service catalog (currently in
+xref:ocp-36-technology-preview[Technology Preview] and not for production
+workloads) during installation or upgrade.
+
+When developing microservices-based applications to run on cloud native
+platforms, there are many ways to provision different resources and share their
+coordinates, credentials, and configuration, depending on the service
+provider and the platform.
+
+To give developers a more seamless experience, {product-title} includes a
+xref:../architecture/service_catalog/index.adoc#architecture-additional-concepts-service-catalog[Service
+Catalog], an implementation of the link:https://openservicebrokerapi.org/[open
+service broker API] (OSB API) for Kubernetes. This allows users to connect any
+of their applications deployed in {product-title} to a wide variety of service
+brokers.
+
+The service catalog allows cluster administrators to integrate multiple
+platforms using a single API specification. The {product-title} web console
+displays the service classes offered by brokers in the service catalog, allowing
+users to discover and instantiate those services for use with their
+applications.
+
+As a result, service users benefit from ease and consistency of use across
+different types of services from different providers, while service providers
+benefit from having one integration point that gives them access to multiple
+platforms.
+
+This feature consists of:
+
+- The Service Consumer: The individual, application , or service that uses a service enabled by the broker and catalog.
+- The Catalog: Where services are published for consumption.
+- Service Broker: Publishes services and intermediates service creation and credential configuration with a provider.
+- Service Provider: The technology delivering the service.
+- Open Service Broker API: Lists services, provisions and deprovisions, binds, and unbinds.
+
+See
+xref:../install_config/install/advanced_install.adoc#enabling-service-catalog[Enabling
+the Service Catalog] for more information.
+
+[[ocp-36-initial-experience]]
+==== Initial Experience (Technology Preview)
+
+In {product-title} 3.6, a better better initial user experience (currently in
```

Comment:
Seems this is new added section after my last review. "better better" duplicates "better"?